### PR TITLE
updates to docker.greymatter.io

### PR DESCRIPTION
**Pull request by chrisbsmith**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Changes the images to pull from `docker.greymatter.io`

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

no additional steps

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

was able to run the following and view the dashboard
```console
make k3d
make credentials
make secrets
make install
```

### CODE REVIEW

File: data/values.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ global:
     path: '/run/spire/socket/agent.sock'
     bind_address: '127.0.0.1'
   # Default configuration for all sidecars in all subcharts
-  sidecar: 
+  sidecar:
     version: '1.4.4'
```
Comment (tilleryd) [inline data/values.yaml:34]: Needs to be 1.4.5

File: fabric/control/values.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ control:
   replicas: 1
   # Version of control
   version: 1.4.4
```
Comment (tilleryd) [inline fabric/control/values.yaml:38]: needs to be 1.4.2

File: data/values.yaml (hunk 1 above)
Comment (chrisbsmith) [reply data/values.yaml]: Updated!

File: fabric/control/values.yaml (hunk 2 above)
Comment (chrisbsmith) [reply fabric/control/values.yaml]: updated!

**Review by justincely — APPROVED** [review]
worked for me in EKS

**Review by tilleryd — APPROVED** [review]

**Pull request merged**